BackofficeBundle/Security/Authorization/Voter/NodeGroupRoleVoter.php

```diff
@@ -90,4 +98,28 @@ public function vote(TokenInterface $token, $object, array $attributes)
 
         return self::ACCESS_ABSTAIN;
     }
+
+    /**
+     * @param NodeInterface  $node
+     * @param GroupInterface $group
+     * @param string         $attribute
+     *
+     * @return boolean
+     */
+    protected function isGrantedNodeGroupRole(NodeInterface $node, GroupInterface $group, $attribute)
+    {
+        $nodeGroupRole = $group->getNodeRoleByNodeAndRole($node->getNodeId(), $attribute);
+        if ($nodeGroupRole instanceof NodeGroupRoleInterface) {
+            if ($nodeGroupRole->isGranted() === 'inherit') {
+                $nodeParent = $this->nodeRepository->findInLastVersion($node->getParentId(), $node->getLanguage(), $node->getSiteId());
+                if ( null !== $nodeParent) {
+                    return $this->isGrantedNodeGroupRole($nodeParent, $group, $attribute);
+                }
+            } else if ($nodeGroupRole->isGranted() === "1") {
```

Comment:
If you put the `1` in a constant, you could put something different (like granted or else)e